src/controllers/commands/CommandController.cpp

```diff
@@ -1355,6 +1355,126 @@ void CommandController::initialize(Settings &, Paths &paths)
             (void)words;  // unused
             return deleteMessages(channel, QString());
         });
+
+    this->registerCommand("/mod", [](const QStringList &words, auto channel) {
+        if (words.size() < 2)
+        {
+            channel->addMessage(makeSystemMessage(
+                "Usage: \"/mod <username>\" - Grant moderator status to a "
+                "user. Use \"mods\" to list the moderators of this channel."));
```

Comment:
```suggestion
                "user. Use \"/mods\" to list the moderators of this channel."));
```
I never noticed this before or I would have fixed it myself

Reply:
This is actually not consistent with what IRC returns, it returns this exact message, without the `/`

Reply:
Well how would "mods" help me Mr. IRC 🤔 
![image](https://user-images.githubusercontent.com/41973452/190937824-6f4b1852-0656-4f36-a5bb-164da0f1d4d4.png)

Reply:
lol, i guess we can fix it regardless